src/lib/agent.c

```diff
@@ -59,9 +59,11 @@ static enum task_state validate(void *data)
 
 	perf_cnt_stamp(&sa->pcd, perf_sa_trace, sa);
 
+#if CONFIG_AGENT_PANIC_ON_DELAY
 	/* panic timeout */
 	if (sa->panic_on_delay && delta > sa->panic_timeout)
 		panic(SOF_IPC_PANIC_IDLE);
+#endif
```

Comment:
I'd suggest to log *error* for `panic_timeout` threshold to have visual difference between *warn* on `warn_timeout` and longer one.